.github/workflows/check.yml

```diff
@@ -9,13 +9,17 @@ on:
 
 jobs:
   check:
-
     runs-on: ubuntu-latest
+
     strategy:
       fail-fast: false
       matrix:
         php-version: [7.2, 7.3, 7.4, 8.0, 8.1]
         protocol: [ 'json', 'msgpack' ]
+        stability: [ '--prefer-dist' ]
```

Comment:
Maybe we can simplify this, 
set 
```
ignorePlatformReq: [' ']
include:
  - php-version: 8.2
    ignorePlatformReq: '--ignore-platform-reqs'
```
and 
```
run: composer install --prefer-dist --no-progress --no-suggest ${{ matrix.ignorePlatformReq}}
```
or 
```
include:
  - php-version: 8.2
    ignorePlatformReq: '--ignore-platform-reqs'
```
and 
```
run: composer install --prefer-dist --no-progress --no-suggest ${{ matrix.ignorePlatformReq}}
```

Reply:
I pushed a commit to try this but the workflow won't run if you add an empty `''` for `ignorePlatformReq`. Can you approve workflow running here please?

Reply:
Ah it did work but my builds weren't running on my fork because GHA for pushes are limited to `main` branch. I temporarily removed that limitation so I can run the tests on GHA. If you approve workflows I can revert that.